site.cfg.example

```diff
@@ -130,6 +130,12 @@
 # include_dirs = /opt/OpenBLAS/include
 # runtime_library_dirs = /opt/OpenBLAS/lib
 
+# [blis]
+# libraries = blis
+# library_dirs = /home/username/blis/lib
+# include_dirs = /home/username/blis/include/blis
```

Comment:
this is unusual, normally the include directory is just "include/" and subfolders are included via the `#include` line